SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -113,63 +128,68 @@ etc...
-1. Set your Unreal project to work with the SpatialOS Unreal GDK fork of the Unreal Engine, which you cloned and installed earlier. To do this:
-    1. In File Explorer, navigate to `<ProjectRoot>\<GameRoot>`.
-    1. Right-click your uproject file and select **Switch Unreal Engine version**.
-    1. Select the path to the Unreal Engine fork you cloned earlier.
-    1. In your terminal, from `Plugins\<UnrealGDKRoot>\SpatialGDK\Build\Scripts`, run `BuildWorker.bat <YourProject>Editor Win64 Development <YourProject>.uproject`
-    1. In File Explorer, right-click your uproject file and select **Generate Visual Studio project files**.
-    1. Open the solution in Visual Studio, and compile and run the project in the Development Editor configuration.
-
-## Running the Schema Generator and setting up the UE4 Editor
-1. In the Unreal Editor, on the SpatialOS Unreal GDK toolbar, click **Generate Schema** to run the `Schema Generator`.
-1. On the same toolbar, click **Snapshot**.
+Set your Unreal project to work with the SpatialOS GDK for Unreal fork of the Unreal Engine, which you cloned and installed earlier. To do this:
+1. In File Explorer, navigate to `<ProjectRoot>\<GameRoot>`.
+1. Right-click your `.uproject` file and select **Switch Unreal Engine version**.
+1. Select the path to the Unreal Engine fork you cloned earlier.
+1. In a terminal window, from `<ProjectRoot>\<GameRoot>\Plugins\UnrealGDK\SpatialGDK\Build\Scripts`, run `BuildWorker.bat <YourProject>Editor Win64 Development <YourProject>.uproject`   
+For Example "`BuildWorker.bat StarterProjectEditor Win64 Development StarterProject.uproject`"  
+This allows you to run locally managed workers. For example running a local spatial deployment with `one_worker` or `two_worker` launch configurations.
+1. In File Explorer, right-click your `.uproject` file and select **Generate Visual Studio project files**.
+1. Open the solution in Visual Studio, and compile and run the project in the Development Editor configuration.
+
+## Running the Schema and Snapshot Generator
+1. In the Unreal Editor, on the SpatialOS Unreal GDK toolbar, click  the **Schema** button to run the `Schema Generator`.
+1. On the same toolbar, click the **Snapshot** button.
 
 ## Running your game
-1. In the Unreal Editor, on the toolbar, open the **Play** drop-down menu and check the box next to **Run Dedicated Server**.
-1. On the SpatialOS Unreal GDK toolbar, click **Launch** and wait for the output `SpatialOS ready`.
+1. In the Unreal Editor, on the toolbar, open the **Play** drop-down menu and check the box next to **Run Dedicated Server** as well as the checkbox for **Spatial Networking**
+1. On the SpatialOS Unreal GDK toolbar, click the green **Launch** (not the default Launch button in the Unreal Editor), this will build your worker configurations (`spatial build build-config`) and `spatial local launch` will be ran in a separate console window, wait for the output "`SpatialOS ready`".
```

Comment:
Orginal:
"1. On the SpatialOS Unreal GDK toolbar, click the green **Launch** (not the default Launch button in the Unreal Editor), this will build your worker configurations (`spatial build build-config`) and `spatial local launch` will be ran in a separate console window, wait for the output "`SpatialOS ready`"."
->
->The `spatial build` and `spatial local launch` don't really 
->
Revised:
"1. Still in the Unreal Editor but this time from the SpatialOS GDK toolbar, select the green **Launch**  button (not the default Launch button from the Unreal Editor toolbar). This builds your [worker configuration]({{URLRoot/content/glossary#worker-configuration) file and launches your game in a local deployment. <br/>
**Launch** opens up a terminal window and runs two SpatialOS command line interface ([CLI]({{URLRoot/content/glossary#cli)) commands: `spatial build build-config` and `spatial local launch`. It is finished when you see `SpatialOS ready` in the terminal window.